pkg/pss/evaluate.go

```diff
@@ -185,7 +356,32 @@ func GetRestrictedFields(check policy.Check) []pssutils.RestrictedField {
 func FormatChecksPrint(checks []pssutils.PSSCheckResult) string {
 	var str string
 	for _, check := range checks {
-		str += fmt.Sprintf("(%+v)\n", check.CheckResult)
+		str += fmt.Sprintf("\n(Forbidden reason: %s, field error list: [", check.CheckResult.ForbiddenReason)
+		for idx, err := range *check.CheckResult.ErrList {
```

Comment:
There doesn't seem to be a final solution for formatting the output as I remember? My previous code just simply formatted it. @realshuting @itsgurmannatsohal